core/app/models/spree/calculator/default_tax.rb

```diff
@@ -1,58 +1,60 @@
 require_dependency 'spree/calculator'
```

Comment:
Shouldn't this require make the change unnecessary?

Reply:
Its not require_dependency'd everywhere its defined/used. 

I'd say we fix the definitions to be consistent and not have to worry about it.